### ISSUE TITLE
Add more online games

### ISSUE THREAD

**Issue by aviaryan** [aviaryan/awesome-no-login-web-apps#136]
I think we can add a lot of good games here. Almost most of the classic games have a good online version.

**Comment by Anshuman-Verma** [comment]
What type of games do we need ? Examples ?

**Comment by Anshuman-Verma** [comment]
Scope of adding games is infinite, we need to narrow down the approach and grab whats suitable for  being in the list.

**Comment by aviaryan** [comment]
The only thing that matters is that we add really *awesome* games, that's all. After that, yes we will still be stuck with a lot of games, in that case I think we should add only the best games from each genre/type (like shooting / racing / fighting and so on ) .

**Comment by code-note** [comment]
Hello, I'm new to OS and would like to contribute here.
Can I add some games from miniclip.com ?

**Comment by aviaryan** [comment]
@code-note Yes, please feel free to do so.

**Comment by aviaryan** [comment]
Resource for good HTML5 games that can work anywhere https://tutorialzine.com/2015/02/30-amazing-games-made-only-with-html5

**Comment by aviaryan** [comment]
@maracuja-juice Awesome game could mean any game which is well-designed and fun to play for a significant population of the human race.

**Comment by maracuja-juice** [comment]
@aviaryan How about [this](https://maracuja-juice.github.io/ultimate-tic-tac-react/) ? :-)

**Comment by aviaryan** [comment]
@maracuja-juice Excellent demonstration of concepts but I would say that an option to play against AI or a live human player would make it more interesting. Right now it's mostly unusable if you are alone.

**Comment by maracuja-juice** [comment]
@aviaryan Yes that is correct. It's something that I would like to work on.

**Comment by herdir** [comment]
Added 3 online games on PR https://github.com/aviaryan/awesome-no-login-web-apps/pull/227

**Comment by aviaryan** [comment]
Thanks for the suggestion!

To add an item to this list, please open a Pull Request with your addition (following the existing README format). Issues requesting new items are being closed to keep the tracker manageable, so please submit a PR if you'd like this to be added.
